Packs/Cybersixgill-DVE/Integrations/CybersixgillDVEEnrichment/CybersixgillDVEEnrichment.yml

```diff
@@ -0,0 +1,65 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Cybersixgill_DVE_Enrichment
+  version: -1
+configuration:
+- display: Sixgill API client ID
+  name: client_id
+  required: true
+  type: 0
+- display: Sixgill API client secret
+  name: client_secret
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  hidden: false
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+description: By enriching CVEs with the DVE Score, Cortex XSOAR customers gain deeper
+  visibility with relevant threat intel from the deep and dark web with dynamic attributes
+  such as where they are trending, POC exploit details, and more. Loaded with extra-context,
+  this allows users to accurately understand the real impact of CVEs to effectively
+  prioritize critical vulnerabilities.
+display: Cybersixgill DVE Enrichment
+name: Cybersixgill_DVE_Enrichment
+script:
+  commands:
+  - arguments:
+    - default: true
+      isArray: true
+      name: cve_id
+      description: 'cves id list'
+      required: true
+      secret: false
+    deprecated: false
+    execution: false
+    description: Returns information for each CVE in the input list
+    name: cybersixgill-cve-enrich
+    outputs:
+      - contextPath: Sixgill.CVE.Overview
+        description: Overview of the CVE enrich information
+        type: String
+      - contextPath: Sixgill.CVE.rawJSON
```

Comment:
Please note that these outputs are being stored as a dict in out context data - which is the incidents cache. therefore, you should store them under a **meaningful** key so you could reach them in a logical way later if they will be needed (if they are not needed, then they shouldn't be stored). Currently, you are storing them all as one dict under `Sixgill.CVE.rawJSON`, and later you will not be able to access each key via demisto. So in order to be able to use them, you should store each under their key (like the value of the CVE is stored under `Sixgill.CVE.value`). There is a simple command for creating the outputs for all the json elements in our demisto-sdk:` json-to-outputs`. See more info here https://xsoar.pan.dev/docs/integrations/context-and-outputs#integration-yml. Plus, if the user wishes to see the whole rae json of the command, her can use a flag raw_response=True when running the command.
Let me know if I can help in any way.